src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -162,11 +160,12 @@ static LoaderResult _parseScene(tvgBlock block, Paint *paint)
 
     if (_paintProperty(block)) return _parsePaintProperty(block, scene);
 
-    if (scene->push(unique_ptr<Paint>(_parsePaint(block))) != Result::Success) {
-        return LoaderResult::MemoryCorruption;
+    if (auto paint = _parsePaint(block)) {
+        scene->push(unique_ptr<Paint>(paint));
```

Comment:
Not mandatory but I suggest add failure checking 
`if (scene->push(unique_ptr<Paint>(paint)) != Result::Success) return LoaderResult::MemoryCorruption;`

Reply:
it's my intention. not so useful.